packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/saved_objects_importer.ts

```diff
@@ -62,7 +62,16 @@ export class SavedObjectsImporter implements ISavedObjectsImporter {
     compatibilityMode,
     managed,
   }: SavedObjectsImportOptions): Promise<SavedObjectsImportResponse> {
-    this.#log.debug('Starting the import process');
+    // Get the allowed types for import from the typeRegistry
+    const allowedTypes = this.#typeRegistry
+      .getImportableAndExportableTypes()
+      .map((type) => type.name);
+    // Log the allowed types, similar to how the exporter logs types
+    this.#log.debug(`Initiating import process.`);
+    this.#log.debug(`Allowed types: [${allowedTypes.join(', ')}]`);
+    this.#log.info(`Import size limit: ${this.#importSizeLimit} saved objects.`);
+    const overwriteStatus = overwrite ? 'enabled' : 'disabled';
+    this.#log.info(`Automatic overwrite is ${overwriteStatus} for the current import operation.`);
```

Comment:
Combine the info logs into one:
```suggestion
    this.#log.info(`Importing with overwrite ${overwriteStatus} and size limit ${this.#importSizeLimit}`);
```

Reply:
Thank you for your detailed feedback @TinaHeiligers, I really appreciate your patience.